.github/workflows/sync-bioconductor.yml

```
@@ -0,0 +1,54 @@
+  workflow_dispatch:     # Allow manual trigger
+
+jobs:
+  sync-from-bioc:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        fetch-depth: 0  # Need full history for rebase
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)